File: Exercise/Hard/03. In N Days/Code/In N Days....js
function calculateDay(startDay, N) {
    const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
    const startIdx = days.indexOf(startDay);

    const endIdx = (startIdx + N) % 7;
    return days[endIdx];
}

let startDay = "Monday";
let N = 4;
let result = calculateDay(startDay, N);
console.log(result);

File: Exercise/Hard/03. In N Days/Code/In N Days....cs
using System;

class Program {
    static void Main() {
        string startDay = Console.ReadLine();

        int N = int.Parse(Console.ReadLine());

        string result = CalculateDay(startDay, N);
        Console.WriteLine(result);
    }

    static string CalculateDay(string startDay, int N) {
        string[] days = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
        int startIdx = Array.IndexOf(days, startDay);

        int endIdx = (startIdx + N) % 7;
        return days[endIdx];
    }
}

